### PR TITLE
Update Finch to v0.7.7

### PR DESCRIPTION
**Pull request by huard**
## Changes

**Non-breaking changes**
- New component version Finch:0.7.7

## Additional Information

Revert "added SENTRY_ENV environment variable" by @matprov in #213
Refresh notebooks by @Zeitsperre in https://github.com/bird-house/finch/pull/201
Add SENTRY_ENV param by @matprov in https://github.com/bird-house/finch/pull/203
Multiple rcps for ensemble processes by @aulemahal in https://github.com/bird-house/finch/pull/202
Remove bugging drop after geoserver upgrade by @aulemahal in https://github.com/bird-house/finch/pull/207
Update to xclim 0.31 by @aulemahal in https://github.com/bird-house/finch/pull/209
Fix xarray, dask and gunicorn related hang by @cjauvin in https://github.com/bird-house/finch/pull/204
run load() before switching to single-threaded scheduler by @huard in https://github.com/bird-house/finch/pull/211
Add average flag for ensemble processes by @aulemahal in https://github.com/bird-house/finch/pull/210
Revert "added SENTRY_ENV environment variable" by @matprov in https://github.com/bird-house/finch/pull/213

### CODE REVIEW

**Review by matprov — APPROVED** [review]

**Review by tlvu — APPROVED** [review]
LGTM.  Let's wait for the Jenkins from DACCS Pipeline result then you can perform the bump2version jazz and merge this PR.  Don't forget to copy/paste the PR description into the merge commit.

Oh, update the CHANGES.md as well.

Same procedure as in https://github.com/bird-house/birdhouse-deploy/pull/195#pullrequestreview-750887404

**Comment by tlvu** [comment]
`finch-usage.ipynb` fixed.  The rest are known (`weaver_example.ipynb`) or intermittent (`WCS_example.ipynb`) errors, this PR is good to go.

Thanks for the quick fix.

File: CHANGES.md [hunk 1]
```diff
@@ -16,6 +16,12 @@
 
 [//]: # (list changes here, using '-' for each new entry, remove this when items are added)
 
+[1.17.5](https://github.com/bird-house/birdhouse-deploy/tree/1.17.5) (2021-11-16)
```
Comment (tlvu) [inline CHANGES.md:19]: You should just added the changes stuff but do not touch this line.  The `bump2version` command will do it for you.

Remove the line above `[//]: # (list changes here, using '-' for each new entry, remove this when items are added)` as well, `bump2version` will restore it for the next change.

**Review by matprov — CHANGES_REQUESTED** [review]
@huard I've spotted an issue while deploying 0.7.6. 

See https://github.com/bird-house/finch/pull/213 for the fix.

**Comment by tlvu** [comment]
@matprov did the deployment failed in the pipeline?  The results are empty.

**Comment by matprov** [comment]
> @matprov did the deployment failed in the pipeline? The results are empty.

@tlvu It's actually caused by the fact that Finch 0.7.7 wasn't released when the infra has been launched.

```
Pulling finch (birdhouse/finch:version-0.7.7)...
manifest for birdhouse/finch:version-0.7.7 not found: manifest unknown: manifest unknown
No containers to restart
executing './components/weaver/post-docker-compose-up'
+ set +o
+ old_state=set +o errexit
set +o noglob
set +o ignoreeof
set +o interactive
set +o monitor
set +o noexec
set +o stdin
set -o xtrace
set +o verbose
set +o vi
set +o emacs
set +o noclobber
set +o allexport
set +o notify
set +o nounset
set +o priv
set +o nolog
set +o debug
+ set +x
Running: ./components/weaver/post-docker-compose-up
  Requested Weaver WPS providers: [
    catalog
    finch
    flyingpigeon
    hummingbird
    malleefowl
    raven
]
  Will retry requests at most for 60s
[Weaver] Wait for response from Magpie to login [https://host-140-88.rdext.crim.ca/magpie]...........................................................
[Weaver] ERROR: Timeout (60s)! Failed to register all providers specified in WEAVER_WPS_PROVIDERS. Magpie is not responding.
```

**Comment by matprov** [comment]
run tests

**Review by tlvu — APPROVED** [review]
LGTM

**Review by matprov — APPROVED** [review]

**Pull request merged**